docs/components/DataList/Web.stories.tsx

```diff
@@ -60,6 +64,13 @@ const Template: ComponentStory<typeof DataList> = args => {
     },
   });
 
+  const [sortingState, setSortingState] = useState<DataListSorting | undefined>(
+    {
+      key: "label",
+      direction: "asc",
+    },
+  );
```

Comment:
maybe we should start undefined on this one since this actually does nothing to the example right now as the test API we use doesn't support that.